Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
     "protocol-units/mempool/*",
     "protocol-units/settlement/mcr/client",
     "protocol-units/settlement/mcr/manager",
+    "protocol-units/maptos-rest",
```

Comment:
Elsewhere, "maptos" is used in crate names because they are supposed to be global, but in the directory tree we don't need the prefix. Rename the dir to something like "api/rest"?